plugins/gatsby-source-wagtail/test/mock.json

```diff
@@ -5,7 +5,44 @@
     "title": "Example site 2",
     "slug": "test-slug-0",
     "last_published_at": "2019-07-16T20:58:09.439440Z",
-    "program_documents": [],
+    "program_documents": {
+      "header": "Program Documents",
+      "display": true,
+      "documents": [
+        {
+          "display_text": "Test Document",
+          "document": "/test/document"
+        },
+        {
+          "display_text": "Test Document 2",
+          "document": "/test/document2"
+        },
+        {
+          "display_text": "Test Document 3",
+          "document": "/test/document3"
+        },
+        {
+          "display_text": "Test Document 4",
+          "document": "/test/document4"
+        },
+        {
+          "display_text": "Test Document 5",
+          "document": "/test/document"
+        },
+        {
+          "display_text": "Test Document 6",
+          "document": "/test/document6"
+        },
+        {
+          "display_text": "Test Document 7",
+          "document": "/test/document7"
+        },
+        {
+          "display_text": "Test Document 8",
+          "document": "/test/document8"
```

Comment:
nit: It's mock data, so it doesn't really matter, but it might be good to have a few with a `url` instead of `document` if we're going to do different icons for them in the future. Also, the paths are going to be absolute paths in the API, not sure if that makes a difference here though.

Reply:
👍 Added some links